hazelcast/src/main/java/com/hazelcast/core/IMap.java

```diff
@@ -189,6 +189,9 @@
      * <p>
      * If you have previously set a TTL for the key, the TTL remains unchanged and the entry will
      * expire when the initial TTL has elapsed.
+     * <p/>
+     * <p><b>Note:</b></p>
```

Comment:
This should just be:
```
<p>
<b>Note:</b> (...)
```
We don't seem to have a fixed style if the text should be in the same line as the `Note`, but at least it should be in the same paragraph. Also these `<p></p>` blocks don't seem to be necessary, a single `<p>` is enough. This should also be Java 8 compliant.

> If you have more than one paragraph in the doc comment, separate the paragraphs with a `<p>` paragraph tag, as shown.

http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format

Reply:
Fixed